MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json

```diff
@@ -0,0 +1,12 @@
+{
+    "risk_rating": "high",
```

Comment:
Why is risk rating high and the Reputation_scan is in the _INFO folder ?

Reply:
thats a new change